src/App.js

```diff
@@ -1,23 +1,101 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useState } from "react";
+import contacts from "./contacts.json";
+import "./App.css";
 
 function App() {
+  const [allContacts, setAllContacts] = useState(contacts.slice(0, 5));
+
+  const randomizeMovie = () => {
+    const randomMovie = contacts[Math.floor(Math.random() * contacts.length)];
```

Comment:
A good idea here is to apply Math.random only on the remaining contacts on the list.
The way you did you might get repeated contacts.